src/lottery/assembly/index.ts

```diff
@@ -7,48 +7,48 @@ type AccountId = string;
 @nearBindgen
 export class Contract {
 
-  private _owner: AccountId
-  private players: PersistentSet<AccountId> = new PersistentSet("p");
-  private _pot: u128 = ONE_NEAR
-  private _active: bool = true
-  private _winner: AccountId = ""
-  private last_played: AccountId = "";
-  private fee_strategy: StrategyType = StrategyType.exponential
+  private owner: AccountId;
+  private players: Set<AccountId> = new Set();
+  private pot: u128 = ONE_NEAR;
+  private active: bool = true;
+  private winner: AccountId;
+  private last_played: AccountId;
+  private fee_strategy: StrategyType = StrategyType.Exponential;
 
   constructor(owner: AccountId) {
-    this._owner = owner;
+    this.owner = owner;
   };
 
   // --------------------------------------------------------------------------
   // Public VIEW methods
   // --------------------------------------------------------------------------
 
-  owner(): AccountId {
```

Comment:
@nearBindgen does not allow for property `get` methods; perhaps it should generate getters for public properties ?

to be consistent with rust interface conventions, changing these methods to `get_XXX()`